examples/cms-sitecore/README.md

```diff
@@ -3,39 +3,44 @@
 This example showcases Next.js's [Static Generation](https://nextjs.org/docs/basic-features/pages) feature using [Sitecore Experience Edge for Content Hub](https://docs.stylelabs.com/contenthub/4.1.x/content/user-documentation/experience-edge/caas-intro.html) as the data source.
 
 # Live Demo
+
 [https://nextjs-blog-sitecore.vercel.app/](https://nextjs-blog-sitecore.vercel.app/)
 
 # Configuration
-## Prerequisites
-To use this project, you need to have your Sitecore Content Hub sandbox up and running with **content publishing enabled**. 
 
-This project uses the content schema and content entities provided **with the Lighthouse Template**. 
+    ## Prerequisites
```

Comment:
Formatting 🙏

Reply:
I am nnot sure what's wrong with the formatting, I already have prettier working / "on save" as well.